kubevirtci

```diff
@@ -168,9 +168,11 @@ function kubevirtci::create_cluster() {
 	export IMAGE_REPO=k8s.gcr.io
 	export CRI_PATH="/var/run/containerd/containerd.sock"
 	template=templates/cluster-template.yaml
-
 	if [ ! -f $template ]; then
-		template="https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/blob/main/templates/cluster-template.yaml"
+		#can't pass the url directly because clusterctl doesn't resolve the redirects properly.
+		mkdir -p templates
```

Comment:
you could do something like `template="$(mktemp -d templateXXXXX)/cluster-template.yaml"`. Then you don't have to worry about whether the "templates" dir already exists or not, and if you're blowing away other info by destroying a templates dir that has other stuff in it.

Reply:
Very nice that is the exact thing I was worried about, didn't know about mktemp.